tests/workflows/basic_subworkflow_deployment/tests/test_workflow.py

```diff
@@ -23,17 +24,28 @@
     ExampleSubworkflowDeploymentNode,
     Inputs,
 )
+from tests.workflows.basic_subworkflow_deployment.workflow_with_base_inputs import (
+    TestInputs,
+    WorkflowWithBaseInputsSubworkflow,
+)
 from tests.workflows.basic_subworkflow_deployment.workflow_with_optional_inputs import (
     InputsWithOptional,
     WorkflowWithOptionalInputsSubworkflow,
 )
 
 
-def test_run_workflow__happy_path(vellum_client):
+@pytest.mark.parametrize(
+    "workflow_class,inputs_class",
+    [
+        (BasicSubworkflowDeploymentWorkflow, Inputs),
+        (WorkflowWithBaseInputsSubworkflow, TestInputs),
```

Comment:
we also need to parameterize the node_class because of the `VellumCodeResourceDefinition.encode(ExampleSubworkflowDeploymentNode)` call below